### ISSUE TITLE
Docs: Feedback for Working with an External Data Store

### ISSUE THREAD

**Issue by mario45211** [hazelcast/hz-docs#362]
Hi, I have some feedback about [this page](https://github.com/hazelcast/hz-docs/edit/main/docs/modules/data-structures/pages/working-with-external-data.adoc)

Could you include note about `NO_TIME_SET` constant in doc ? I was confused at first time when I haven't set a TTL on entry but EntryStore returned `Long.MAX_VALUE`.

Thanks in advance!

**Comment by mario45211** [comment]
My case:
1. Using EntryStore in my implementation
2. Forgot to explicitly set TTL on Map's Entry during put/set operation on that map
3. In EntryStore.save execution I was confused why entry has set TTL 
4. Discovered that TTL **in Hazelcast's code** is by default set using `NO_TIME_SET` constant, which I've used to detect entry without TTL

I found that mentioning about that constant in documentation could save time of other developers, which could fall into the same case as mine.
